salt/modules/cassandra_cql.py

```diff
@@ -336,13 +336,13 @@ def cql_query(query, contact_points=None, port=None, cql_user=None, cql_pass=Non
             query = query[cluster_version]
         except KeyError:
             query = query.get(major, max(query))
-        log.debug('New query is: {0}'.format(query))
+        log.debug('New query is: %s', query)
 
     try:
         results = session.execute(query)
     except BaseException as e:
-        log.error('Failed to execute query: {0}\n reason: {1}'.format(query, str(e)))
-        msg = "ERROR: Cassandra query failed: {0} reason: {1}".format(query, str(e))
+        log.error('Failed to execute query: %s\n reason: %s', query, e)
+        msg = "ERROR: Cassandra query failed: {0} reason: {1}".format(query, six.text_type(e))
```

Comment:
We don't need `six.text_type()` here since we're passing the exception to `.format()` which will do that for us automagically.